### PR TITLE
Created topic: Connect to Nebula Graph

### PR DESCRIPTION
**Pull request by randomJoe211**

### CODE REVIEW

**Review by whitewum — DISMISSED** [review]

**Review by whitewum — APPROVED** [review]

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+| [nebula-java](https://github.com/vesoft-inc/nebula-java)               | The official Python client of Nebula Graph             | [Graph Client Example](https://github.com/vesoft-inc/nebula-java#graph-client-example)                                                                    |
+
+If you don't have a Nebula Graph database yet, we recommend that you try our cloud service. [Nebula Graph Cloud Service](https://www.nebula-cloud.io/) supports on-demand and deployment and fast building, and uses Nebula Graph Studio as its default client.
+
+## Use nebula-console to connect to Nebula Graph
+
+1. Download [the nebula-console binary file](TODO) [Coming soon].
+
+    ```bash
+    wget [Coming soon]
```
Comment (whitewum) [inline docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md:22]: add bash indicater: 
```bash
> wget [coming soon]
```

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    ```
+
+2. Use the binary file to connect to Nebula Graph.
+
+    `./nebula-console [-addr IP] [-port port] -u <username> -p <password> [-t 120] [-e "an nGQL command" | -f filename.nGQL]`
```
Comment (whitewum) [inline docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md:27]: add bash indicator

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+## Nebula-console export mode
+
+When the export mode is enabled, nebula-console exports all the query results into a CSV file. When the export mode is disabled, the export stops. The syntax is as follows.
+
+> **NOTE**: The following commands are case insensitive.
+
+* Enable nebula-console export mode:
+
+```nGQL
+:SET CSV <your_file.csv>
```
Comment (whitewum) [inline docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md:49]: add nebula indictor:
```
(root@nebula) [(none)]>> :SET CSV <your_file.csv>
```

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+* Disable nebula-console export mode:
+
+```nGQL
+:UNSET CSV
```
Comment (whitewum) [inline docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md:55]: ditto

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
+# Connect to Nebula Graph
+
+Nebula Graph supports multiple types of clients, including a CLI client, a GUI client, and clients developed in popular programming languages. This topic provides an overview of Nebula Graph clients and a basic instruction on how to use the native CLI client, nebula-console.
```
Comment (Amber1990Zhang) [inline docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md:3]: Are CLI client and nebula-console the same? I think it's not clear here

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+## Nebula Graph clients
+
+| Client                                                                 | Introduction                                           | Connection guide                                                                                                                                          |
+| ---------------------------------------------------------------------- | ------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [nebula-console](https://github.com/vesoft-inc/nebula-console)         | The native command-line interface of Nebula Graph      | [Connect to Nebula Graph with nebula-console](#use-nebula-console-to-connect-to-nebula-graph)                                                             |
+| [Nebula Graph Studio](https://github.com/vesoft-inc/nebula-web-docker) | The official graphical user interface for Nebula Graph | [Connect to Nebula Graph with Nebula Graph Studio](https://github.com/vesoft-inc/nebula-web-docker/blob/master/docs/nebula-graph-studio-user-guide-en.md) |
```
Comment (Amber1990Zhang) [inline docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md:10]: Unify the description with Nebula Graph Studio. @oldLady344

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+| [nebula-go](https://github.com/vesoft-inc/nebula-go)                   | The official Golang client of Nebula Graph             | [nebula-go code example](https://github.com/vesoft-inc/nebula-go#usage-example)                                                                           |
+| [nebula-python](https://github.com/vesoft-inc/nebula-python)           | The official Python client of Nebula Graph             | [How to use nebula-python in your code](https://github.com/vesoft-inc/nebula-python#how-to-use-nebula-python-in-your-code)                                |
+| [nebula-java](https://github.com/vesoft-inc/nebula-java)               | The official Python client of Nebula Graph             | [Graph Client Example](https://github.com/vesoft-inc/nebula-java#graph-client-example)                                                                    |
+
+If you don't have a Nebula Graph database yet, we recommend that you try the cloud service. [Nebula Graph Cloud Service](https://www.nebula-cloud.io/) supports on-demand and deployment and fast building, and uses Nebula Graph Studio as its default client.
```
Comment (Amber1990Zhang) [inline docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md:15]: supports on-demand deployment and fast building

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+1. Download [the nebula-console binary file](TODO) [Coming soon].
+
+    ```bash
+    $ wget [Coming soon]
+    ```
+
+2. Use the binary file to connect to Nebula Graph.
+
+    ```bash
+    $ ./nebula-console [-addr IP] [-port port] -u <username> -p <password> [-t 120] [-e "an_nGQL_command" | -f filename.nGQL]
```
Comment (Amber1990Zhang) [inline docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md:28]: what's the difference between [] and <>. I think [] means optional.

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+    | -addr        | Set the IP address of the graphd service. The default address is 127.0.0.1.                                                                                                   |
+    | -port        | Set the port number of the graphd service. The default port number is 3699.                                                                                                   |
+    | -u/-user     | Set the username of your Nebula Graph account. The default username is root.                                                                                                  |
+    | -p/-password | Set the password of your Nebula Graph account. The default password for the root user is nebula.                                                                              |
+    | -t/-timeout  | Set an integer-type timeout threshold of the connection. The unit is second. The default value is 120.                                                                        |
+    | -e/-eval     | Set a string-type nGQL command. The nGQL command is executed once the connection succeeds. You'll get the return message and the connection stops then.             |
```
Comment (Amber1990Zhang) [inline docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md:39]: nGQL statement
avoid future tense

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md (hunk 5 above)
Comment (randomJoe211) [reply docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md:3]: Yes, they are the same. The first sentence shows that Nebula Graph supports several types of client. Your point being?

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md (hunk 6 above)
Comment (randomJoe211) [reply docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md:10]: No need. Here is only a very brief description. The link is provided if anyone wants to know more.

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md (hunk 8 above)
Comment (randomJoe211) [reply docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md]: Fair enough. addr and port should not be optional.

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+    | -port        | Set the port number of the graphd service. The default port number is 3699.                                                                                                   |
+    | -u/-user     | Set the username of your Nebula Graph account. The default username is root.                                                                                                  |
+    | -p/-password | Set the password of your Nebula Graph account. The default password for the root user is nebula.                                                                              |
+    | -t/-timeout  | Set an integer-type timeout threshold of the connection. The unit is second. The default value is 120.                                                                        |
+    | -e/-eval     | Set a string-type nGQL statement. The nGQL statement is executed once the connection succeeds. The connection stops after the result is returned..             |
+    | -f/-file     | Set the path of an nGQL file. The nGQL statements in the file are executed once the connection succeeds. You'll get the return messages and the connection stops then. |
```
Comment (oldLady344) [inline docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md:41]: all the option name must be in code format

in the Description column, use third-person singular verbs to start the sentences.

**Review by whitewum — APPROVED** [review]

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+    | -addr        | Set the IP address of the graphd service. The default address is 127.0.0.1.                                                                                                   |
+    | -port        | Set the port number of the graphd service. The default port number is 3699.                                                                                                   |
+    | -u/-user     | Set the username of your Nebula Graph account. The default username is root.                                                                                                  |
+    | -p/-password | Set the password of your Nebula Graph account. The default password for the root user is nebula.                                                                              |
+    | -t/-timeout  | Set an integer-type timeout threshold of the connection. The unit is second. The default value is 120.                                                                        |
+    | -e/-eval     | Set a string-type nGQL statement. The nGQL statement is executed once the connection succeeds. The connection stops after the result is returned..             |
```
Comment (oldLady344) [inline docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md:40]: result is returned..  ==》result is returned.

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+
+```nGQL
+(root@nebula) [(none)]> :QUIT
+
+Bye root!
+```
+
+## FAQ
+
+TODO [This section should include connection-related FAQs. When users met a problem on connection, they should first come here for a solution. If the content gets stacked, remove this section to a single topic.]
```
Comment (oldLady344) [inline docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md:73]: users met a problem on ==》 users meet a problem on

File: docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md (hunk 10 above)
Comment (randomJoe211) [reply docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md:41]: It is not necessary to use the code format for table headers, whether they are options or parameters. But it is good to have a standard style.

**Review by Amber1990Zhang — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Pull request merged**